rust/ql/test/library-tests/type-inference/type-inference.ql

```diff
@@ -1,18 +1,60 @@
 import rust
+import utils.test.InlineExpectationsTest
 import codeql.rust.internal.TypeInference as TypeInference
 import TypeInference
-import utils.test.InlineExpectationsTest
 
 query predicate inferType(AstNode n, TypePath path, Type t) {
   t = TypeInference::inferType(n, path)
 }
 
-query predicate resolveMethodCallExpr(MethodCallExpr mce, Function f) {
-  f = resolveMethodCallExpr(mce)
+module ResolveTest implements TestSig {
+  string getARelevantTag() { result = ["method", "fieldof"] }
+
+  private predicate functionHasValue(Function f, string value) {
+    f.getPrecedingComment().getCommentText() = value
+    or
+    not exists(f.getPrecedingComment()) and
+    value = f.getName().getText()
```

Comment:
Perhaps add a TODO comment to use canonical name instead (if it exists)?

Reply:
Done 👍